tests/utils/shared_helpers.py

```diff
@@ -0,0 +1,518 @@
+from simpeg import utils
+import numpy as np
+
+
+class ProlateEllispse:
```

Comment:
Fixing a typo here, but also making a proposal for renaming this class:
```suggestion
class ProlateEllipsoid:
```